src/docs/product/profiling/differential-flamegraphs.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Differential Flame Graphs
+sidebar_order: 110
+description: "Learn how to use and interpret differential flamegraphs"
+---
+
+There are multiple ways to visualize and use profiling data, including differential flamegraphs, which can help easily identify function regressions or changes in the execution context.
+
+## What Are Differential Flamegraphs?
+
+Differential flamegraphs are graph visualizations that highlight how the call stack frequencies of a program have changed, a task which can be tedious to figure out without them. They're often used in analysis for finding function regressions and understanding which functions got slower, faster, or were newly added. Keep in mind that differential flamegraphs aren't really a new concept, but just aggregate flamegraphs, drawn using the colors generated by the diff of the two.
+
+In Sentry, differential flamegraphs are shown whenever a function regression is detected. When a regression happens, Sentry generates two aggregate flamegraphs. One from the period leading up to the regression, and another from the time right after it. A differential flamegraph is then used to visualize the data.
+
+<Note>
+
+In a differential flamegraph, the color red means that a function's execution time has increased or that there's a newly present function. The color blue means that the execution time has decreased.
+
+</Note>
+
+Be on the lookout for red frames when interpreting differential flamegraphs. They're the reason why your application code has been performing worse.
+
+![Differential flamegraph lock contention](./differential-flamegraph.png)
+
+The differential flamegraph above is showing that the function call to `ModuleLock.acquire` has increased in duration, hinting at a lock contention problem which is slowing down the code.
+
+Let's look at another example:
+
+![Differential flamegraph new functions](./differential-flamegraph-new.png)
+
+The above differential flamegraph shows that new function calls to `OrganizationTeamsEndpoint.dispatch` were introduced, which made the code run slower, while calls to `ChunkUploadEndpoint.get` decreased.
+
+If you look closely, you'll notice that the top right corner has a `Before -> After` and `After -> Before` toggle. This is because by default, differential flamegraphs draw the aggregate flamegraph from after a regression has occured as the source of truth. This means that any code that may have been removed, will no longer be drawn (after all, it's not there anymore). This is why negating the view is useful, as it allows us to compare the data from before and peek into the future of how our code will change.
+
```

Comment:
```suggestion

The top left corner has a `Before -> After` and `After -> Before` toggle. This is because it's helpful to be able to view data from **before a regression occurred** and compare it with data after the regression. This is a different view than a differential flamegraph, which shows the aggregate flamegraph drawn right **after a regression has occurred**. 

```
@JonasBa let me know if this makes sense :)